bot/exts/info/tags.py

```diff
@@ -373,38 +303,68 @@ async def get_command(self, ctx: Context, *, argument_string: Optional[str]) ->
+                    self.accessible_tags(interaction.user),
+                    interaction, Embed(title="Available tags"),
+                    **self.PAGINATOR_DEFAULTS,
                 )
             else:
-                await ctx.send(embed=Embed(description="**There are no tags!**"))
+                await interaction.response.send_message(embed=Embed(description="**There are no tags!**"))
             return True
 
-        identifier = TagIdentifier.from_string(argument_string)
+        identifier = TagIdentifier.from_string(tag_name)
 
         if identifier.group is None:
             # Try to find accessible tags from a group matching the identifier's name.
-            if group_tags := self.accessible_tags_in_group(identifier.name, ctx.author):
+            if group_tags := self.accessible_tags_in_group(identifier.name, interaction.user):
                 await LinePaginator.paginate(
-                    group_tags, ctx, Embed(title=f"Tags under *{identifier.name}*"), **self.PAGINATOR_DEFAULTS
+                    group_tags, interaction, Embed(title=f"Tags under *{identifier.name}*"), **self.PAGINATOR_DEFAULTS
                 )
                 return True
 
-        embed = await self.get_tag_embed(ctx, identifier)
+        embed = await self.get_tag_embed(interaction, identifier)
         if embed is None:
             return False
 
         if embed is not COOLDOWN.obj:
             await wait_for_deletion(
-                await ctx.send(embed=embed),
-                (ctx.author.id,)
+                await interaction.response.send_message(embed=embed),
+                (interaction.user.id,)
             )
         # A valid tag was found and was either sent, or is on cooldown
         return True
 
+    @get_command.autocomplete("tag_name")
+    async def tag_name_autocomplete(
+        self,
+        interaction: discord.Interaction,
+        current: str
+    ) -> list[app_commands.Choice[str]]:
+        """Autocompleter for `/tag get` command."""
+        tag_names = [tag.name for tag in self.tags.keys()]
+        return [
+            app_commands.Choice(name=tag, value=tag)
+            for tag in tag_names if current.lower() in tag
+        ]
+
+    @tag_group.command(name="list")
+    async def list_command(self, interaction: discord.Interaction) -> bool:
+        """Lists all accessible tags."""
+        if self.tags:
+            await LinePaginator.paginate(
+                self.accessible_tags(interaction.user),
+                interaction,
+                Embed(title="Available tags"),
+                **self.PAGINATOR_DEFAULTS,
+            )
+        else:
+            await interaction.response.send_message(embed=Embed(description="**There are no tags!**"))
+        return True
+
 
 async def setup(bot: Bot) -> None:
     """Load the Tags cog."""
     await bot.add_cog(Tags(bot))
+    await bot.tree.sync(guild=discord.Object(id=GUILD_ID))
```

Comment:
>I don't think any of the syncing code should be necessary, since we do that automatically now (https://github.com/python-discord/bot-core/pull/171)

@wookie184 should this also be removed?

Reply:
Even after removing this, the bot works fine. So I guess its best to remove it.

Reply:
I believe we should be able to remove this too, yeah, since there's nothing specific to the cog.